docs/examples/amor.ipynb

```diff
@@ -129,19 +129,34 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pl[TutorialFilename[Sample]] = \"amor2023n000608.hdf\"\n",
```

Comment:
After discussion on Slack: maybe this is a case where traditional looping would be less painful?
We are already caching the computation of the reference manually, so we could do
```Py
runs = {
    "amor2023n000608.hdf": sc.scalar(0.85, unit='deg'),
    "amor2023n000609.hdf": sc.scalar(2.25, unit='deg'),
    "amor2023n000610.hdf": sc.scalar(3.65, unit='deg'),
}

results = {}
for file, angle in runs.items():
    pl[TutorialFilename[Sample]] = file
    pl[SampleRotation[Sample]] = angle
    results[file] = pl.compute(NormalizedIofQ).hist()

sc.plot(results, norm='log', vmin=1e-4)
```
?

Unless using cyclebane would mean that we could remove the manual reference caching step?

Reply:
In practice we will read the reference from a file anyway, so it doesn't make much sense to have the reference reduction be part of the pipeline that reduces samples.

I'm okay with doing it the 'traditional way' here.